cmd/semaphore/config/arguments.go

```diff
@@ -60,8 +62,26 @@ func ConstructArguments(params *Semaphore) ([]config.Option, error) {
 		arguments = append(arguments, semaphore.WithServices(protobuffers.ServiceResolver(params.Protobuffers, path)))
 	}
 
+	log.Println()
+	log.Println()
+	log.Println(params.HTTP)
+	log.Println()
+	log.Println()
+
 	if params.HTTP.Address != "" {
-		arguments = append(arguments, semaphore.WithListener(http.NewListener(params.HTTP.Address, specs.Options{})))
+		arguments = append(
+			arguments,
+			semaphore.WithListener(
+				http.NewListener(
+					params.HTTP.Address,
+					http.WithOrigins(params.HTTP.Origin),
+					http.WithReadTimeout(params.HTTP.ReadTimeout),
+					http.WithWriteTimeout(params.HTTP.WriteTimeout),
+					http.WithKeyFile(params.HTTP.KeyFile),
+					http.WithCertFile(params.HTTP.CertFile),
```

Comment:
The options could be passed but are not yet configurable. Please include these options inside the HCL provider and parse the options.

Reply:
it was already done :thinking:

Reply:
HCL syntax is defined over here:
https://github.com/jexia/semaphore/blob/master/pkg/providers/hcl/intermediate.go#L31

Options fetched over here:
https://github.com/jexia/semaphore/blob/master/pkg/providers/hcl/hcl.go#L106

Reply:
My bad! I thought I did not see the change.